.storybook/preview.js

```diff
@@ -5,15 +5,26 @@ import { vue3SourceDecorator } from "./decorators/vue3SourceDecorator";
 
 // Vue plugins
 import { createVueless } from "vueless";
+import { createRouter, createWebHistory } from "vue-router";
 
 // Tailwind styles
 import "./index.pcss";
 
 // Create vueless instance
 const vueless = createVueless();
 
+const router = createRouter({
+  history: createWebHistory(import.meta.env.VITE_BASE_PATH),
+  routes: [
```

Comment:
format this code `routes: [...]` in 1 line and add this to storybook preset package.